app/src/services/fetchArticles/fetchArticles.ts

```diff
@@ -0,0 +1,16 @@
+import {ArticlesResponse} from 'types/articles';
+
+const fetchArticles = async (
+  searchQuery?: string
+): Promise<ArticlesResponse> => {
+  const withQueryParam = searchQuery ? `?q=${searchQuery}` : '';
+  const response = await fetch(`http://0.0.0.0:8000/v1/news${withQueryParam}`);
```

Comment:
I would definitely recommend using dotenv for API endpoints as it helps keep our application more secure.